core/src/ibc/connection_03/manager.rs

```diff
@@ -221,6 +221,12 @@ impl<'a> Manager<'a> {
         Some(connection_end)
     }
 
+    pub fn query_client_connections(&self, identifier: &str) -> ConnectionIdentifiersInClient {
```

Comment:
Let's make all `Manager`'s query functions to be private.

Reply:
I fixed them.